app/src/main/java/org/kiwix/kiwixmobile/utils/files/FileSearch.java

```diff
@@ -92,27 +96,27 @@ public FileSearch(Context ctx, ResultListener listener) {
     return book;
   }
 
+  /**
+   * Uses RxJava Completable to scan the file system and media store concurrently.
+   * Synchronizes the check and finally calls onScanCompleted after completion.
+   * @param defaultPath
```

Comment:
You should either explain this parameter or remove this line.

Reply:
Ok
Any clue on the build failure? I can't see any stacktrace on Travis CI.

Reply:
We run some Espresso tests on Tesdroid. On some device(s) one or more tests have failed. You should always run Espresso test on your local devices before making a PR.